ui/admin/app/styles/app.scss

```diff
@@ -598,69 +598,71 @@
   }
 }
 
-.ordered-series-diagram {
-  --group-title-line-height: 2.25rem;
+.ordered-series-diagram-wrapper {
+  min-height: 150px;
```

Comment:
For these larges ones I think we make exceptions? @randallmorey is that the general rule?

Reply:
I think we can keep all units the same and convert this to rem, 9 or 10 rem could work here

Reply:
@DhariniJeeva totally agree with you.  There are a few instances in which we use container height and width as px but each fix.